### ISSUE TITLE
[Bun] Module importing error

### ISSUE THREAD

**Issue by 8549** [twlite/youtube-sr#56]
Hello, I am using Bun 1.0.13 and youtube-sr 4.3.9, importing with `import { YouTube } from "youtube-sr";` gives the following error:
```
Could not find a declaration file for module 'youtube-sr'. '[REDACTED]/node_modules/youtube-sr/dist/mod.mjs' implicitly has an 'any' type.
There are types at '[REDACTED]/node_modules/youtube-sr/dist/mod.d.ts', but this result could not be resolved when respecting package.json "exports". The 'youtube-sr' library may need to update its package.json or typings.ts(7016)
```
What is the correct way to import the library using Bun?
Thanks in advance.

**Comment by twlite** [comment]
Hi, this seems to be caused by the exports field in package.json file, I will fix this soon.

**Comment by 8549** [comment]
Thanks, any ETA on the fix release? How can I import the library in the meantime?

**Comment by twlite** [comment]
@8549 hi this should be fixed now with v4.3.10
